packages/config/src/projects/layer2s/edgeless.ts

```diff
@@ -31,21 +36,67 @@ export const edgeless: Layer2 = underReviewL2({
     },
     activityDataSource: 'Blockchain RPC',
   },
-  transactionApi: {
-    type: 'rpc',
-    startBlock: 1,
-    defaultUrl: 'https://rpc.edgeless.network/http',
-    defaultCallsPerMinute: 1500,
-    assessCount: subtractOne,
-  },
-  badges: [Badge.VM.EVM, Badge.Stack.Orbit, Badge.RaaS.Caldera],
-  escrows: [
+  rpcUrl: 'https://rpc.edgeless.network/http',
+  badges: [Badge.VM.EVM, Badge.DA.CustomDA, Badge.RaaS.Caldera],
+  nonTemplateEscrows: [
     // this is not the bridge escrow itself but the strategy contract that holds all funds backing the ewETH in the canonical bridge escrow. The normal escrow can be used as soon as we track the ewETH token
     {
       address: EthereumAddress('0xbD95aa0f68B95e6C01d02F1a36D8fde29C6C8e7b'),
       sinceTimestamp: new UnixTime(1711057199),
-      tokens: '*',
+      tokens: ['ETH', 'stETH'],
```

Comment:
These two will not appear on FE unless you fetch them from discovery, idk if you want them to